recipes/ml_dtypes/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "ml_dtypes" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ml_dtypes-{{ version }}.tar.gz
+  sha256: d19cde6fb503b7258e8edeb895fa66b99bf9fcb6fcffb4becfae71d4890016c7
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - python
```

Comment:
We plan to follow NEP-29 upstream
https://numpy.org/neps/nep-0029-deprecation_policy.html
which means the minimum Numpy we will test with upstream right now will be 1.21, and after next month we will at some point bump the minimum Python version to 3.9.

Looks good otherwise!

That's not to say it won't work against older versions, but we don't plan to test it.